### ISSUE TITLE
"Error parsing version list JSON:illegal value" MultiMc 5

### ISSUE THREAD

**Issue by qqrcoisa** [MultiMC/MultiMC5#1848]
System Information
-----------------------------
MultiMC version: 5

Operating System: linux mint selena. Oracle/java 8

Summary of the issue or suggestion:
----------------------------------------------
Does not allow loading any Forge version. It shows this error block saying "Error parsing version list JSON:illegal value" and then it shows a black block saying no forge versions are currently available for minecraft *whatever version*.
I tried deleting the Json files, as advised by someone in a forum, it did not work. I formated my computer to get the multimc working again, which it did, for a few days, and then this error appeared again. 
 
What should happen:
------------------------------
It shold load the forge version for the minecraft version selected in order to create the new instance and allow us to load the game with the chosen mod.

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1. Open a new file on MultiMC

2. install the minecraft version desired and load chosen mod.

3. try installing forge and error appears.

Suspected cause:
---------------------------

I have absolutley no idea. I read the json files and it seems to be only concerning the internet conection and log in.

Logs/Screenshots:
----------------------------

I am not sure what to paste here, I do apologise. Will be happy to be guided if necessary.

Additional Info:
---------------------------
Did not find any usable information online regarding this matter.

**Comment by peterix** [comment]
So what is inside those json files *exactly*?

**Comment by qqrcoisa** [comment]
Hi peterix,

Thanks for replying. As much as I would love to copy/paste the file and the list.jason here, I deleted. In that forum, the only one I've found refering to the same issue, some guy managed to fix his problem by deleting it, so I did the same. But it showed many red marked lines referring to the internet provider and the attempnts to log in. I have then marked the pacjage (multimc) to be completly removed, but never got a clean instalation. It picked up from where we've left before removing, thus all files and instances we had before, also the removed files not there. I know this is not helpfull at all, so I'll format again, reinstall, and see if we get the same issue again and the post it. Many thanks, we do love multimc, its awesome to be able to count on it.
